docs/guides/query-kyc-state-on-c-chain.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_position: 45
+title: Query KYC State on C-Chain
+description: This document describes how to use Remix IDE to query KYC State of an address on C-Chain
+---
+
+# How to Query KYC State on C-Chain
+
+This document provides a guide on how to use Remix IDE to query the KYC state of
+an address on C-Chain. Follow the steps below to accomplish this.
+
+## Requirements
+
+- MetaMask installation and functionality
+
+## Add Camino Network to MetaMask
+
+To query the KYC state of an address using the Remix IDE, we will use the MetaMask wallet.
+
+We need to add a custom network to MetaMask. In this document, we will use the Columbus test network.
```

Comment:
just to be sure this is was done with the new Columbus network, correct? 
Some partners have been using the old test network, so might be helpful to specify.

Reply:
This is hard to specify and the old testnet will be gone. So let's keep it like this.